app/controllers/report_controller.rb

```diff
@@ -10,37 +10,22 @@ def rsolr_request_error(exception)
   end
 
   def data
-    params[:sord] ||= "asc"
-    rows_per_page = params[:rows] ? params.delete(:rows).to_i : 10
-    params[:per_page] = rows_per_page * [params.delete(:npage).to_i, 1].max
+    params[:per_page] = params[:per_page] ? params[:per_page].to_i : Report::ROWS_PER_PAGE
+    params[:page] ||= 1
     @report = Report.new(params, current_user:)
 
     respond_to do |format|
       format.json do
         render json: {
           page: params[:page].to_i,
-          records: @report.num_found,
-          total: (@report.num_found / rows_per_page.to_f).ceil,
-          rows: @report.report_data
+          last_page: (@report.num_found / params[:per_page].to_f).ceil,
+          data: @report.report_data
```

Comment:
These changes align the server-side impl with what the client-side wants.

Reply:
ditto with code comment.

Reply:
Done.